upgrade/upgrade-scripts/upgrade-container

```diff
@@ -351,6 +356,7 @@ function create_upgrade_container() {
 		[Files]
 		PrivateUsersChown=no
 		Bind=/dev/zfs
+		Bind=/sys/module
```

Comment:
Can this be read only? Is it R/W on 6.0 right now?

Reply:
I've just tested and it is currently R/W on 6.0.
I could test to see if things still work with `BindReadOnly` instead of `Bind`.

Reply:
Seems like that should work. I've opened DLPX-78329 so that can be investigated further.

Reply:
Thanks. What you landed is consistent with what we're previously doing, so that's good with me.